src/content/plugins/split-chunks-plugin.md

```diff
@@ -0,0 +1,220 @@
+
+W> When assigning equal names to different split chunks they are merged together. This can be used to put all vendor modules into a single shared chunk, but it's not recommend since it can result in more code downloaded.
+
+The magic value `true` automatically chooses a name based on chunks and cache group key. Elsewise a string or function can be passed.
+
+When the name matches an entrypoint name, the entrypoint is removed.
+
+### Select chunks
+
+With the `chunks` option the selected chunks can be configured. There are 3 values possible `"initial"`, `"async"` and `"all"`. When configured the optimization only selects initial chunks, on-demand chunks or all chunks.
+
+The option `reuseExistingChunk` allows to reuse existing chunks instead of creating a new one when modules match exactly.
+
+This can be controlled per cache group.
+
+### Select modules
+
+The `test` option controls which modules are selected by this cache group. Omitting it selects all modules. It can be a RegExp, string or function.
+
+It can match the absolute module resource path or chunk names. When a chunk name is matched, all modules in this chunk are selected.
+
+## `optimization.splitChunks`
+
+This configuration object represents the default behavior of the `SplitChunksPlugin`.
+
+```js
+splitChunks: {
+	chunks: "async",
+	minSize: 30000,
+	minChunks: 1,
+	maxAsyncRequests: 5,
+	maxInitialRequests: 3,
+	name: true,
+	cacheGroups: {
+		default: {
+			minChunks: 2,
+			priority: -20
+			reuseExistingChunk: true,
+		},
+		vendors: {
+			test: /[\\/]node_modules[\\/]/,
+			priority: -10
+		}
+	}
+}
+```
+
+By default cache groups inherit options from `splitChunks.*`, but `test`, `priority` and `reuseExistingChunk` can only be configured on cache group level.
+
+`cacheGroups` is an object where keys are the cache group names. All options from the ones listed above are possible: `chunks`, `minSize`, `minChunks`, `maxAsyncRequests`, `maxInitialRequests`, `name`.
+
+You can set `optimization.splitChunks.cacheGroups.default` to `false` to disable the default cache group.
+
+The priority of the default groups are negative to allow any custom cache group to take higher priority (the default value is `0`).
+
+Here are some examples and their effect:
+
+### Example 1
+
+Create a `commons` chunk, which includes all code shared between entrypoints.
+
+```js
+splitChunks: {
+	cacheGroups: {
+		commons: {
+			name: "commons",
+			chunks: "initial",
+			minChunks: 2
+		}
+	}
+}
+```
+
+W> This downloads more code than neccessary.
+
+### Example 2
+
+Create a `vendors` chunk, which includes all code from node_modules in the whole application.
+
+``` js
+splitChunks: {
+	cacheGroups: {
+		commons: {
+			test: /[\\/]node_modules[\\/]
+			name: "vendors",
+			chunks: "all"
+		}
+	}
+}
+```
+
+W> This downloads more code than neccessary.
+
+## `optimization.runtimeChunk`
+
+Setting `optimization.runtimeChunk` to `true` adds an additonal chunk to each entrypoint containing only the runtime.
```

Comment:
include the different things that you can pass to runtimeChunk. It can be a boolean, `'single'`, `'multiple'` or and object that has property `name` whose value can be either a string or a function of signature `(entrypoint: Chunk) => string`

Reply:
Hey @sokra can you fill in some technical gap here? Should we add `enforce` details in this piece, I would need your help with it 😓

Reply:
What is its default value? In this [blog post](https://medium.com/webpack/webpack-4-mode-and-optimization-5423a6bc597a) it says it's `true`, but from trying it out it's not.